mobile/Screen/MapScreen.js

```diff
@@ -1,39 +1,265 @@
-import React from 'react';
+import React, { useEffect, useState, useRef } from 'react';
 import {
-  Button, StyleSheet, Text, View,
+  StyleSheet,
+  Text,
+  View,
+  Animated,
+  Image,
+  Dimensions,
 } from 'react-native';
-import axios from 'axios';
-// eslint-disable-next-line import/no-unresolved
-import { URL } from '@env';
+import MapView, { Marker } from 'react-native-maps';
+
+const { width, height } = Dimensions.get('window');
+
+const CARD_HEIGHT = height / 4;
+const CARD_WIDTH = CARD_HEIGHT + 50;
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: '#fff',
+  },
+  scrollView: {
+    position: 'absolute',
+    bottom: 30,
+    left: 0,
+    right: 0,
+    paddingVertical: 10,
+  },
+  endPadding: {
+    paddingRight: width - CARD_WIDTH,
+  },
+  card: {
+    padding: 10,
+    borderRadius: 8,
+    elevation: 2,
+    backgroundColor: '#FFF',
+    marginHorizontal: 10,
+    shadowColor: '#000',
+    shadowRadius: 5,
+    shadowOpacity: 0.3,
+    shadowOffset: { x: 2, y: -2 },
+    height: CARD_HEIGHT,
+    width: CARD_WIDTH,
+    overflow: 'hidden',
+  },
+  cardImage: {
+    flex: 3,
+    borderRadius: 8,
+    width: '100%',
+    height: '100%',
+    alignSelf: 'center',
+  },
+  textContent: {
+    flex: 1,
+  },
+  cardTitle: {
+    fontSize: 12,
+    marginTop: 5,
+    fontWeight: 'bold',
+  },
+  cardDescription: {
+    fontSize: 12,
+    color: '#444',
+  },
+  markerWrap: {
     alignItems: 'center',
     justifyContent: 'center',
   },
+  marker: {
+    width: 20,
+    height: 20,
+    borderRadius: 5,
+    backgroundColor: 'rgba(130,4,150, 0.9)',
+  },
+  ring: {
+    width: 55,
+    height: 55,
+    borderRadius: 12,
+    backgroundColor: 'rgba(130,4,150, 0.3)',
+    position: 'absolute',
+    borderWidth: 1,
+    borderColor: 'rgba(130,4,150, 0.5)',
+  },
 });
 
-function MapScreen() {
-  const test = async () => {
-    console.log(URL);
-    try {
-      const result = await axios.post(`${URL}/test/post`, {
-        name: 'James', age: 20,
-      });
-      console.log(result.data);
-    } catch (err) {
-      console.error(err);
-    }
+const Images = [
+  { uri: 'http://overcomeandamplify.com/wp-content/uploads/2016/09/broadway-mall.jpg' },
+  { uri: 'https://img.ctykit.com/cdn/ca-dtla/images/tr:w-1800/king-eddy.jpg' },
+  { uri: 'https://images1.apartments.com/i2/KYlRC-2yNZXQSNh0JGMno_S-RIbHbS7bDROWLLtZ1uc/111/star-apartments-los-angeles-ca-primary-photo.jpg' },
+  { uri: 'https://www.lehrerarchitects.com/wp-content/uploads/2017/12/JAMES-WOOD_06-1920x1100.jpg' },
+];
+
+function GalleryScreen() {
```

Comment:
Don't know what happened here, but somehow this changed to `GalleryScreen` when it's `MapScreen`?